client/src/components/Header/Header.js

```diff
@@ -18,4 +18,5 @@ const FixedNav = styled.div`
   margin: 0 auto;
   font-family: 'AirbnbCereal_W_Bk', 'Montserrat', sans-serif, Courier, monospace;
   border-bottom: 1px solid rgb(230, 230, 230);
+  z-index: 9999;
```

Comment:
with this z-index, the nav menu goes under the nav.
please set it below the nav menu's z-index

Reply:
It definitely works, do you think it’s to high for 9999

Reply:
<img width="318" alt="Screenshot 2023-06-22 at 2 26 14 PM" src="https://github.com/czhaoca/TheKimsPlusTwo/assets/67996710/b00e5441-e982-4064-9614-a4787b184dd3">

Reply:
We'll merge this to main. Thank you :)

Reply:
We may change that block's z-index to flip it on the top. Or using a hamburg menu style.

Reply:
@chris718 Hi christina while you guys are working on that
Could you please check if nav is working ok on smaller window size?

![image](https://github.com/czhaoca/TheKimsPlusTwo/assets/104462860/2277beae-98f6-4b46-b1d7-29efca5186f8)

Reply:
It looks like this.
<img width="516" alt="Screenshot 2023-06-22 at 7 45 03 PM" src="https://github.com/czhaoca/TheKimsPlusTwo/assets/68087157/b7f68871-5651-4c80-a8bf-3759179cbebc">